aws/cloudformation-templates/base/_template.yaml

```diff
@@ -102,6 +102,52 @@ Parameters:
     Type: String
     Description: The GitHub branch used for deployment.
 
+  FenixZipDetectUrl:
+    Type: String
+    Description: Fenix Commerce Zipcode Detect URL
+    Default: 'https://ipapi.co/json?key=cKGC3jQbSIoXYmI2KtXObugsKfosD9Yr0HnkHhPUu1SM2wQhE0'
+
+  FenixTenantId:
+    Type: String
+    Description: Fenix Commerce Demo Env TenantID
+    Default: 'ec6ea3439489426ba09cf6c906ead8d5'
+    NoEcho: true
+
+  FenixEddEndpoint:
+    Type: String
+    Description: Fenix Commerce Estimated Delivery Date Endpoint URL
+    Default: 'https://awsretaildemo.delest.fenixcommerce.com/fenixdelest/api/v2/deliveryestimates'
+
+  FenixMonetaryValue:
+    Type: String
+    Description: Fenix Commerce Default monetary value
```

Comment:
add a description if it's important for the demo. If we don't expect the customer to change it, have it as a hardcoded parameter to the web-ui service but not the main tpl